README.md

```diff
@@ -1,27 +1,38 @@
-## AI for Litter Detection (Web Application)
+# Trash AI: Web application for serverless image classification of trash
+[![Website](https://img.shields.io/badge/Web-TrashAI.org-blue)](https://www.trashai.org)
+
 
 ### Project Information
 
--   Sponsor: Win Cowger, UC Riverside - Trash Data Projects
--   Meeting Times: Wednesdays at 6:30pm (Weekly Brigade Meetings)
+-   Sponsor: Win Cowger, [Moore Institute for Plastic Pollution Research](https://mooreplasticresearch.org/)
+-   Meeting Times: Wednesdays at 6:30pm PT [Weekly Brigade Meetings](https://www.meetup.com/code4sac/)
 
 ### Project Summary
 
--   Description: Develop web application where users can upload photos of street litter which will be labeled using computer vision to detect and categorize litter type.
--   Benefit: Enhance abilities of researchers to quickly label photos and categorize types.
+Trash AI is a web application where users can upload photos of litter which will be labeled using computer vision to detect and categorize litter in the image by type. Early inspiration from [WADE AI](https://github.com/letsdoitworld/wade-ai) streamlined this development. Trash AI will enhance the abilities of researchers to quickly label trash in photos.
```

Comment:
First sentence--place comma after "litter" and before "which".

Reply:
updated! Thanks for the recommendation.